### PR TITLE
Feature/#9 question model, closes #9

### LINKED ISSUE #9: As a developer, I want to have the model of a question represented in the backend of my application
AC:
- Database entity exists
- REST API for CRUD Operations on this entity exists

_Hint_
-Don't think to far in advance when it comes to designing the schema of the entity, as later migrations will handle later extension of functionality

### PR DESCRIPTION
**Pull request by m0e33**
This PR adds the question model.

Testing steps:
- auto generated testing cases

Acceptance Criteria:
- Database entity exists
- REST API for CRUD Operations on this entity exists

Solves issue #9 .

Steps needed for migration:
- rails db:migrate RAILS_ENV=development

### CODE REVIEW

File: spec/views/questions/show.html.erb_spec.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe "questions/show", type: :view do
```
Comment (phoeinx) [inline spec/views/questions/show.html.erb_spec.rb]: Testing a route that doesn't exist anymore doesn't make that much sense :D

File: spec/views/questions/edit.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+RSpec.describe "questions/edit", type: :view do
+  before(:each) do
+    @question = assign(:question, Question.create!(
+      :author => "MyText",
+      :content => "MyText"
+    ))
+  end
+
+  it "renders the edit question form" do
```
Comment (phoeinx) [inline spec/views/questions/edit.html.erb_spec.rb]: Also fails because of missing views

File: spec/controllers/questions_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+# removed from Rails core in Rails 5, but can be added back in via the
+# `rails-controller-testing` gem.
+
+RSpec.describe QuestionsController, type: :controller do
+
+  # This should return the minimal set of attributes required to create a valid
+  # Question. As you add validations to Question, be sure to
+  # adjust the attributes here as well.
+  let(:valid_attributes) {
+    skip("Add a hash of attributes valid for your model")
```
Comment (phoeinx) [inline spec/controllers/questions_controller_spec.rb]: When we actually have input fields to test this should be filled in.

File: .gitignore [hunk 4]
```diff
@@ -11,6 +11,8 @@ capybara-*.html
 *.orig
 rerun.txt
 pickle-email-*.html
+.idea/
```
Comment (vxrahn) [inline .gitignore]: Please don't include IDE-specific configuration.

**Pull request closed without merging**